src/api/adamik/broadcast.ts

```diff
@@ -8,6 +8,13 @@ export type BroadcastResponse = {
   error?: { message: string };
 };
 
+export type BackendErrorResponse = {
```

Comment:
This is the same for others API calls, so it could be put in _src/utils/types.ts_ and used for other API calls too.